bench-vortex/src/bin/clickbench.rs

```diff
@@ -96,10 +99,37 @@ fn main() -> anyhow::Result<()> {
         None => Builder::new_multi_thread().enable_all().build(),
     }
     .expect("Failed building the Runtime");
-    let basepath = format!("clickbench_{}", args.flavor).to_data_path();
-    let client = reqwest::blocking::Client::default();
 
-    args.flavor.download(&client, basepath.as_path())?;
+    let url = match args.use_remote_data_dir {
+        None => {
+            let basepath = format!("clickbench_{}", args.flavor).to_data_path();
+            let client = reqwest::blocking::Client::default();
+
+            args.flavor.download(&client, basepath.as_path())?;
+            Url::parse(
+                ("file:".to_owned() + basepath.to_str().vortex_expect("path should be utf8") + "/")
+                    .as_ref(),
+            )
+            .unwrap()
+        }
+        Some(remote_data_dir) => {
+            // e.g. "s3://vortex-bench-dev-eu/parquet/"
```

Comment:
If you could change the github action we have to regenerate the tpch data to also include these that would be great

Reply:
Yea makes sense will add